src/compiler/diagnosticMessages.json

```diff
@@ -4280,5 +4284,13 @@
     "Remove all unused labels": {
         "category": "Message",
         "code": 95054
+    },
+    "Convert to use async and await":{
+        "category": "Message",
+         "code": 95055

```

Comment:
There is an extra space here

Reply:
I think I'd prefer "Convert to async function", in case anyone else does.